src/components/List.js

```diff
@@ -138,7 +138,9 @@ const List = ({ token }) => {
   return (
     <div>
       <h1>List</h1>
-
+      {/* <Typography variant="h1" component="h2">
```

Comment:
Should this commented code be deleted before merging?

Reply:
Yes, I missed that one. Thanks!